### PR TITLE
changes

### PR DESCRIPTION
**Pull request by maxjakob**
Trying to see the diff in ReviewNB

### CODE REVIEW

File: sklearn-demo.ipynb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
```
Comment (arnaudselva) [inline sklearn-demo.ipynb:16]: <p>this is a comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (maxjakob) [inline sklearn-demo.ipynb:16]: <p>I guess these comments are per cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (maxjakob) [inline sklearn-demo.ipynb:16]: <p>small data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (arnaudselva) [reply sklearn-demo.ipynb:16]: yep per cell

Comment (arnaudselva) [reply sklearn-demo.ipynb:16]: <p>yes per cell</p>

Comment (maxjakob) [reply sklearn-demo.ipynb:16]: <p>replying to a comment</p>

Comment (se-jaeger) [inline sklearn-demo.ipynb:16]: <p>I think this is wrong. Should be 0.815</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (se-jaeger) [inline sklearn-demo.ipynb:16]: <p>Hmm, unfortunately it's not possible to tell you to only delete this <code># keep two dimensions two plot</code>  comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (maxjakob) [reply sklearn-demo.ipynb:16]: <p>ability to add emojis is dearly missed here</p>

Comment (maxjakob) [reply sklearn-demo.ipynb:16]: <p>right, I also found it a bit coarse-grained that comments are per cell.</p>

Comment (tripl3a) [reply sklearn-demo.ipynb:16]: <p>if only someone would write a tool that recognizes or <strong>ignores</strong> such <em>cross-cell</em>  changes.</p><p> </p><blockquote>sdfdsf</blockquote><p> </p><p><code>sdfdsfd</code></p><p> </p><p><code><a href="https://www.ecosia.org" target="_blank">sdfdsf</a></code></p><p> </p><h1>dfsdf</h1><h2>sfdsdf</h2><h3>sdfsd</h3>

**Comment by maxjakob** [comment]
This was fun :) Closing.

File: sklearn-demo.ipynb (hunk 1 above)
Comment (maxjakob) [inline sklearn-demo.ipynb:16]: <pre class="ql-syntax" spellcheck="false">Line #5.    np.random.seed(0)
Line #6.    cov1 = np.array([[0., -0.1], [1.7, .4]])
</pre><p>This is how you can comment on lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/#comment-1354780052'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Pull request closed without merging**